habitat-hitl/habitat_hitl/core/client_message_manager.py

```diff
@@ -335,10 +313,15 @@ def update_navmesh_triangles(
             ]
 
     def update_camera_transform(
```

Comment:
Maybe `set_camera_transform` would be better? I notice in the Unity PR that if you don't update on a given frame, then the viewport doesn't get rendered at all, is that right? I can imagine a user setting up a PIP to show a static overhead view, and then wondering why it isn't showing up.

Reply:
Good point.

I'm unsure whether this is self-explanatory due to the fact that most messages are immediate-mode. It's definitely worth a docstring update.